docs/insomnia/declarative-config.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: article-detail
+title:  Declarative Config
+category: "Work with Other Kong Products"
+category-url: kong-products
+---
+
+{:.alert .alert-primary}
+**Note**: This feature is currently only available on Design Documents.
+
+Insomnia creates a preformatted [decK](https://docs.konghq.com/deck) file when you add endpoints to a Document.
```

Comment:
Insomnia generates a [Declarative Config](https://docs.konghq.com/gateway-oss/2.5.x/db-less-and-declarative-config/#loading-the-declarative-configuration-file) file, from the OpenAPI spec you write in the Design tab. Users can then use `decK` to push that declarative config into their Kong instance (but you don't necessarily need `decK` for it, you can do it directly too: https://docs.konghq.com/gateway-oss/2.5.x/db-less-and-declarative-config/#loading-the-declarative-configuration-file.